MusicBandSimulator/MusicBandSimulator.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32407.343
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MusicBandSimulator", "MusicBandSimulator\MusicBandSimulator.csproj", "{2949B1CF-D344-4360-9607-E554B20F8E82}"
```

Comment:
Why project identifier is 'FAE04EC0-301F-11D3-BF4B-00C04F79EFBC'? Is it mistake?

Reply:
i don't give a fuck what are you talking about, man

Reply:
that's a rofl, man, take it easy) the shit big M mentioned is called a project Identifier - unique value to identify proj file, it's auto-generated whenever you're creating a new project